controllers/redis/redis_controller.go

```diff
@@ -70,7 +71,11 @@ func New(mgr manager.Manager) (*RedisReconciler, error) {
 	if err != nil {
 		return nil, err
 	}
-	providerList := []providers.RedisProvider{awsRedisProvider, openshift.NewOpenShiftRedisProvider(client, logger)}
+	providerList := []providers.RedisProvider{
+		openshift.NewOpenShiftRedisProvider(client, logger),
+		awsRedisProvider,
+		gcp.NewGCPRedisProvider(client),
```

Comment:
same as two previouse comments - could we validate Openshift and GCP BlobStorage Providers same way as awsPostgresProvider. And also - error to be informative.